meta-mender-commercial/README.md

```diff
@@ -1,3 +1,3 @@
-This layer is for using Mender commercial components. There has not been an
-official release of these yet, so more information will be available at a later
-time.
+Mender 2.2 introduced Enterprise features. Please email
```

Comment:
Late to the game, but should this say `Mender 2.2 introduced commercial features` as they are not exclusive to Enterprise plan and would be available for Professional plan as well.

Reply:
Ok, fixed in [followup](https://github.com/mendersoftware/meta-mender/commit/d6ad3ec69030366f30f5c15f3f7cb7f69b8d370b) and cherry-picked to warrior and thud as well.